lms/js_config_types.py

```diff
@@ -118,8 +118,11 @@ class DashboardRoutes(TypedDict):
     students: str
     """Paginated endpoint to fetch students"""
 
-    assignment_grades_sync: str
-    """Sync grades for a given assignment"""
+    assignment_grades_sync: str | None
```

Comment:
We could use a different approach to expose the flag, ie explicitly expose the flag in the config here as a boolean.

I don't have a preference.